index.js

```diff
@@ -8,11 +8,33 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+const travelerDiv = document.querySelector('div#profile')
+travelerDiv.querySelector('img').src = `${traveler.photo}`
+travelerDiv.querySelector('h2').textContent = `${traveler.name}` 
+travelerDiv.querySelector('em').textContent = `${traveler.nickname}` 
+
 
 
 /***** Deliverable 4 *****/
 
+traveler.animalSightings.forEach(obj => {
+    const li = document.createElement('li')
+    li.dataset.id = obj.id
+
+    li.innerHTML = `
+        <p>${obj.description}</p>
+        <img src="${obj.photo}" alt="${obj.species}"/>
+        <a href="${obj.link}" target="_blank">Here's a video about the ${obj.species} species!</a>
+    `
+
+    document.querySelector('ul#animals').append(li)
+
+})
```

Comment:
Excellent work here! Nice use of creating and appending elements, and also using innerHTML. Creating and appending elements is actually more efficient in terms of time and space complexity, so keep that in mind for optimization in the future. Both ways are perfectly fine, however.